Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -151,11 +151,11 @@ function Initialize-DevelopmentEnvironment {
 
         [Parameter(Mandatory=$false)]
         [ValidateSet('4.0.0', '5.0.0')]
-        [String] $StandardVersion = '5.0.0',
+        [String] $StandardVersion,
```

Comment:
We will need to update https://techdocs.ed-fi.org/display/ODSAPIS3V71/Getting+Started+-+Source+Code+Installation to include standard and extension version on the initialization steps